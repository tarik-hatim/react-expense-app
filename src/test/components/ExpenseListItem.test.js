import React from 'react';
import { shallow } from 'enzyme';
import ExpenseItem from '../../components/ExpenseListItem';
import expenses from '../fixtures/expenses';

test('Should render an expense item', () => {
    const wrapper = shallow(<ExpenseItem {...expenses[0]} />);
    expect(wrapper).toMatchSnapshot();
});