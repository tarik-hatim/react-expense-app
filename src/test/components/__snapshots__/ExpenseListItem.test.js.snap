// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render an expense item 1`] = `
<div>
  <Link
    to="/edit/1"
  >
    <h3>
      Gum
    </h3>
  </Link>
  <p>
    $180.00
    -
    January 1st 1970
  </p>
</div>
`;
